user/forms.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.forms import UserCreationForm
+from .models import User
+
+class SignUpForm(UserCreationForm):
+   
+    class Meta:
+        model = User
+        fields = ('username', 'email', 'birthday')
+
+        labels = {
+            'username': 'ユーザネーム',
+            'email': 'email',            
```

Comment:
細かい質問ですが、以下のようにしていないのは意図的ですか？
```suggestion
            'email': 'メールアドレス',            
```